test/analytics_fetcher/test_makebulk.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from analytics_fetcher.makebulk import page_info_docs, path_components
+
+
+class TestMakebulk(unittest.TestCase):
+    def test_single_component(self):
+        self.assertEqual(path_components('/fred'), ['/fred'])
+
+    def test_multiple_components(self):
+        self.assertEqual(path_components('/fred/wilma'), ['/fred', '/fred/wilma'])
+
+    def test_page_info_docs_with_single_element(self):
```

Comment:
Looks good! I am wondering if either this test should be renamed to `test_page_info_docs` or keep that name but also include another test for multiple elements? Otherwise I am not sure if the single element naming has significance? 

```
    def test_page_info_docs_with_multiple_elements(self):
        traffic = {
            "/fred/wilma": { 1: [1, 10, 0.1] }
        }
        expected_action = {
            "index": {"_type": "page-traffic", "_id": '/fred/wilma'}
        }
        expected_data = {
            "path_components": ['/fred', '/fred/wilma'],
            "rank_1": 1,
            "vc_1": 10,
            "vf_1": 0.1
        }

        results = list(page_info_docs(traffic))
        self.assertEqual(expected_action, results[0][0])
        self.assertEqual(expected_data, results[0][1])
```

However maybe having single and multiple is overkill and testing the `path_component` again?

Reply:
Thanks @Rosa-Fox - I might create a `multiple` test to see if it adds anything. If not, I'll rename this test as you suggest.

Reply:
Hi @Rosa-Fox - as this method only works on a single traffic entry, I've updated the comments in the first commit with an example input/output and renamed the test as you've suggested. Thanks.

Reply:
👍